### DISCUSSION TITLE
38

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#800] [Comments]
http://jojoldu.tistory.com/38

**Comment by jojoldu** [comment]
작성자: seungdols

창천향로님 블로그를 feed로 받아 구독중인 신입 개발자로써 저에겐 이런 고뇌가 참으로 반가운것 같습니다. 
제가 볼때는 높아보이는 분께서도 이런 고민을 하시니 '약간은 인간적인 면모를 보는 것' 같아서요. 
항상 응원하겠습니다. 너무 많이 남과 비교 하지 않으셔도 될 것 같습니다. (충분히 높은 분이신 것 같습니다.)

**Comment by jojoldu** [comment]
작성자: 창천향로

헉;; 거의 일기장처럼 쓰고 있어서 누군가가 feed를 할것이라곤 생각못했네요 ㅠㅠ 
저도 아직 한참 모자란 주니어 개발자입니다^^;; seungdols님도 잘하시는것 같습니다! (블로그를 방문했었습니다^^;) 격려 너무 감사드립니다!

**Comment by jojoldu** [comment]
작성자: 낭군이

okjsp에서 리눅스 책 후기 보고 온 블로그인데
좋네요:) 남하고 너무 비교하지 마세유

**Comment by kosomi** [comment]
rrrrrrrr
